### ISSUE TITLE
Varia & Varia Child Themes: Accessibility related issues in Comments

### ISSUE THREAD

**Issue by itowhid06** [Automattic/themes#2043]
Screen readers can’t read `<b>` tag content. So, when some important text is wrapped in `<b>` tags, it will come to no benefit for users using screen readers. We should use `<strong>` tags instead.

## Steps to replicate

1. Go to any posts with comments.
2. Inspect any comment author's name in Devtools.
3. See the author's name is wrapped in a `<b>` tag.

## Result

![image](https://user-images.githubusercontent.com/12055657/82280553-eac08200-99b0-11ea-86f4-3d02b5aaad0d.png)

As evident in the above screenshot the author's name is wrapped in a `<b>` tag. This is hapenning because, though there is a Comment Walker class named as `Varia_Walker_Comment` it is not used in `wp_list_comments` function. As a result the aforementioned function is using `Walker_Comment` Walker class from WordPress Core, which is producing those `<b>` tags. As mentioned in [https://core.trac.wordpress.org/ticket/50202](https://core.trac.wordpress.org/ticket/50202), `<b>` tags should be avoided in favor of `<strong>` tags for Accessibility reasons. Note that, there is also an extra badge printing after the comment author's name when `Varia_Walker_Comment` walker is used.

## Expected

 The author's name in a comment should be wrapped in a `<strong>` tag as evident in many core themes like **Twenty Nineteen** & **Twenty Twenty** themes.  Also Varia theme should use `Varia_Walker_Comment` walker.

**Comment by jffng** [comment]
Thanks @itowhid06! Based on the discussion in the core ticket you linked above, it's not clear to me that `<strong>` needs to be used here for screen readers. 

So, I don't think introducing the comment walker here is worth the effort to refactor the styling and test across all Varia + child themes.

That said, we should remove the comment walker class from Varia as it is not used.
